components/card/stories/template.js

```diff
@@ -87,7 +87,7 @@ export const Template = ({
                   Icon(
                     {
                       size: "xxl",
-                      iconName: showAsset === "folder" ? "File" : "Document",
+                      iconName: showAsset === "folder" ? "Folder" : "Document",
```

Comment:
"File" isn't an icon (apparently), but "Folder" is. Which actually makes more sense when `showAsset: "folder" 😆

Reply:
Nice! Let's hope that translates to our S2 icons too when the time comes.